pkg/apiserver/profiling/protobuf_to_svg.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
+
+package profiling
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"strconv"
+	"time"
+
+	"github.com/goccy/go-graphviz"
+	"github.com/google/pprof/driver"
+	"github.com/google/pprof/profile"
+)
+
+func convertProtobufToSVG(content []byte, task TaskModel) ([]byte, error) {
+	dotContent, err := convertProtobufToDot(content, task)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert protobuf to dot: %v", err)
+	}
+	svgContent, err := convertDotToSVG(dotContent)
+	if err != nil {
+		return nil, fmt.Errorf("failed to convert dot to svg: %v", err)
+	}
+
+	return svgContent, nil
+}
+
+func convertProtobufToDot(content []byte, task TaskModel) ([]byte, error) {
+	args := []string{
+		"-dot",
+		// prevent printing stdout
+		"-output", "dummy",
+		"-seconds", strconv.Itoa(int(1)),
+	}
+	address := fmt.Sprintf("%s:%d", task.Target.IP, task.Target.Port)
+	args = append(args, address)
```

Comment:
I suspect this address might not be needed. Maybe it can be any address?

Reply:
Yes, you are right. The value of address is not used here, but the arg is still required by driverPporf to convert protobuf to dot format. We can also set it to an empty string.